src/iconsManifest/supportedExtensions.ts

```diff
@@ -4335,6 +4335,8 @@ export const extensions: IFileCollection = {
         '.yarn-metadata.json',
         '.yarnignore',
       ],
+      filenamesGlob: ['.yarnrc'],
```

Comment:
Not sure about this change. According to #2714 we would need support for `.yarnrc.yml` as a folder. Is that right? And it seems also that we should add `.yarnrc.yml` as filename. If that's the case, why using globs and not just add ` .yarnrc.yml` to the collection? It would be clearer.

Reply:
@KingDarBoja any comments about this?

Reply:
I thought I answered this before... guess I never submitted anything lol

Anyway, the `.yarnrc.yml` should be used for yarn v2 and onwards, so there is no need to use the filenamesGlob as you said.